packages/common-all/src/vault.ts

```diff
@@ -4,7 +4,11 @@ import { normalizeUnixPath } from ".";
 import { CONSTANTS } from "./constants";
 import { DendronError } from "./error";
 import { DVault, WorkspaceFolderRaw } from "./types";
+import { QuickPickItem } from "vscode";
```

Comment:
you can't import `QuickPickItem` in `common-all` because it doesn't have access to vscode. it only works in development because of the monorepo